diff --git a/vitest.config.ts b/vitest.config.ts
index 0868870e..a8011625 100644
--- a/vitest.config.ts
+++ b/vitest.config.ts
@@ -6,6 +6,8 @@ export default defineConfig({
   plugins: [vue()],
   resolve: {
     alias: {
+      '@/custom': resolve(__dirname, '../../overlay/custom/client'),
+      '@custom': resolve(__dirname, '../../overlay/custom/client'),
       '@': resolve(__dirname, 'packages/client/src'),
       electron: resolve(__dirname, 'tests/mocks/electron.ts'),
       '/logo.png': resolve(__dirname, 'packages/client/public/logo.png'),
    },
