diff --git a/packages/client/src/router/index.ts b/packages/client/src/router/index.ts
index abb0497b..ef81ce47 100644
--- a/packages/client/src/router/index.ts
+++ b/packages/client/src/router/index.ts
@@ -36,7 +36,7 @@ const router = createRouter({
     {
       path: '/hermes/cockpit',
       name: 'hermes.cockpit',
-      component: () => import('@/custom/cockpit/views/CockpitView.vue'),
+      component: () => import('@/custom/ia2/views/scenes/CollabScene.vue'),
       meta: { fullscreen: true },
       children: [
         {
