diff --git a/packages/client/src/i18n/locales/en.ts b/packages/client/src/i18n/locales/en.ts
index 3d6baf82..cacafa52 100644
--- a/packages/client/src/i18n/locales/en.ts
+++ b/packages/client/src/i18n/locales/en.ts
@@ -268,6 +268,7 @@ export default {
 
   // Common
   common: {
+    close: 'Close',
     loading: 'Loading...',
     cancel: 'Cancel',
     delete: 'Delete',
diff --git a/packages/client/src/i18n/locales/zh.ts b/packages/client/src/i18n/locales/zh.ts
index 957dc892..20805b26 100644
--- a/packages/client/src/i18n/locales/zh.ts
+++ b/packages/client/src/i18n/locales/zh.ts
@@ -268,6 +268,7 @@ export default {
 
   // 通用
   common: {
+    close: '关闭',
     loading: '加载中...',
     cancel: '取消',
     delete: '删除',
